dev/functional/cxx_core_features.h

```diff
@@ -70,3 +70,7 @@
 #if __cpp_concepts >= 201907L
 #define SQLITE_ORM_CONCEPTS_SUPPORTED
 #endif
+
+#if __cpp_lib_three_way_comparison >= 201907L
+#define SQLITE_ORM_THREE_WAY_COMPARISON_SUPPORTED

```

Comment:
what is three way comparison?

Reply:
a new operator in C++ 20:   <=>, which replaces the need to define <,<=,>,>=,!=,== if you provide it! The compiler can also provide it for you if you use =default.
It is a dependency of the chronos library and sys_day requires C++ 20, but there is no __cpp_lib_sysday so I used the feature testing 3 way comparison instead